docs/docs/2-guides/3-supported-apis.mdx

```diff
@@ -49,12 +49,14 @@ import {ConsulAPIBadge} from "@site/src/components/CustomBadge";
 |                  | Node Service List                 | GET, /v1/catalog/node-services/:node           |   ✅    |
 | Coordinates      | Read WAN Coordinates              | GET /v1/coordinates/datacenters                |   ✅    |
 |                  | Read LAN Coordinates              | GET /v1/coordinates/nodes                      |   ✅    |
+|                  | Read LAN Coordinates for a node   | GET /v1/coordinate/node/:node                  |   ✅    |
 | Events           | Fire Event                        | PUT /v1/event/fire/:name                       |   ✅    |
 |                  | List Events                       | GET /v1/event/list                             |   ✅    |
 | Health           | List Checks for Node              | GET /v1/health/node/:id                        |   ✅    |
 |                  | List Checks for Service           | GET /v1/health/checks/:id                      |   ✅    |
 |                  | List Nodes for Service            | GET /v1/health/service/:id                     |   ✅    |
 |                  | List Checks in State              | GET /v1/health/state/:state                    |   ✅    |
+|                  | List Nodes for connect-capable    | GET /v1/health/connect/:service                |   ✅    |
```

Comment:
I think it is ok, add longer entries here. It is going to be rendered correctly on the webpage anyway:
```suggestion
|                  | List Nodes for connect-capable  service | GET /v1/health/connect/:service                |   ✅    |
```

Reply:
Great!! Done